src/app/features/edit-course/edit-course.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CoursesStoreService } from '@app/services/courses-store.service';
+import { MappingService } from '@app/services/mapping.service';
+import { CourseInfo } from '@app/shared/types/course.model';
+import { combineLatest, Subscription, take } from 'rxjs';
+
+@Component({
+  selector: 'app-edit-course',
+  templateUrl: './edit-course.component.html',
+  styleUrls: ['./edit-course.component.css'],
+})
+export class EditCourseComponent implements OnInit, OnDestroy {
+  courseId: string | undefined;
```

Comment:
Just like the other comment, this can be disregarded.

Reply:
Done